erp/mapper/vaccination.py

```diff
@@ -0,0 +1,211 @@
+from datetime import datetime
+
+from django.contrib.gis.geos import Point
+
+from core.lib import text
+from erp.models import Accessibilite, Commune, Erp
+from erp.provider import arrondissements
+
+
+class RecordMapper:
+    FIELDS_MAP = {
+        "c_nom": "nom",
+        "c_adr_num": "numero",
+        "c_adr_voie": "voie",
+        "c_com_nom": "commune",
+        "c_com_cp": "code_postal",
+        "c_com_insee": "code_insee",
+        "c_rdv_tel": "telephone",
+    }
+
+    RESERVE_PS = [
+        "R\u00e9serv\u00e9 aux professionnels de sant\u00e9",
+        "Uniquement pour les professionnels de sant\u00e9",
+        "Ouvert uniquement aux professionnels",
+    ]
+
+    def __init__(self, record, today=None):
+        self.erp = None
+        self.today = today if today is not None else datetime.today()
+        try:
+            self.geometry = record["geometry"]
+            self.props = record["properties"]
+        except KeyError as err:
+            raise RuntimeError(f"Propriété manquante {err}: {record}")
+        # Clean string properties
+        for key, val in self.props.items():
+            self.props[key] = text.strip_if_str(val)
+
+    @property
+    def source_id(self):
+        value = self.props.get("c_gid")
+        if value is None:
+            raise RuntimeError("Champ c_gid manquant")
+        return str(value)
+
+    @property
+    def erp_exists(self):
+        return self.erp and self.erp.id is not None
+
+    def process(self, activite):
+        "Procède aux vérifications et à l'import de l'enregistrement"
+        # Création ou récupération d'un ERP existant
+        self.__fetch_or_create_erp(activite)
+        self.__check_importable()
+        self.__import_basic_erp_fields()
+        self.__import_coordinates()
+        # Commune checks and normalization
+        self.__retrieve_commune_ext()
+        # Strange/invalid phone numbers
+        if self.erp.telephone and len(self.erp.telephone) > 20:
+            self.erp.telephone = None
+        # Build metadata
+        self.erp.metadata = self.__build_metadata()
+        # Prepare comment
+        commentaire = self.__build_commentaire()
+
+        # Save erp instance
+        self.erp.save()
+
+        # Attach a comment to the Accessibilite object
+        if self.erp_exists and self.erp.has_accessibilite():
+            accessibilite = self.erp.accessibilite
+        else:
+            accessibilite = Accessibilite(erp=self.erp)
+        accessibilite.commentaire = commentaire
+        accessibilite.save()
+
+        return self.erp
+
+    def __build_commentaire(self):
```

Comment:
I'd rather use single underscore for "privates" methods, see also: https://stackoverflow.com/a/6930223